python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -674,6 +672,8 @@ class RandomForestClassifier(Base):
                               threshold, algo,
                               num_classes, convert_dtype,
                               fil_sparse_format, predict_proba):
+        if num_classes != self.num_classes:
```

Comment:
not required within the `_predict_model_on_gpu` function as we are passing `self.num_classes` to the `num_classes` variable.